src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -174,19 +257,25 @@ def __find_free_process(self):
                 except TimeoutExpired:
                     continue
 
-    def __update_process(self, pid:int, log_file, device):
+    def __update_process(self, pid: int, log_file, device):
```

Comment:
align types

Reply:
sorry, didn't get you, do you suggest refactoring the function signature?

Reply:
I mean, we should make a note to align function args to one style.
```
def __update_process(self, pid: int, log_file: type, device: str):
```

anyway is minor and can not be a blocker

Reply:
not a part of this PR